src/pages/mypage/components/AcornHistory.tsx

```
@@ -0,0 +1,159 @@
+import { useEffect, useMemo, useRef, useState } from "react";
+import IcAcorn from "@/assets/icons/ic_acorn.svg?react";
+import IcPlus from "@/assets/icons/ic_plus.svg?react";
+
+function IcMinus({ className }: { className?: string }) {
+  return (
+    <svg
+      className={className}
+      viewBox="0 0 16 16"
+      fill="none"
+      aria-hidden
+    >
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)